tests/target/hex_literal_ignore.rs

```diff
@@ -0,0 +1,5 @@
+// rustfmt-hex_literal_case: Ignore
```

Comment:
Please also rename this file to account for the `ignore` --> `preserve` change
```suggestion
// rustfmt-hex_literal_case: Preserve
```